coapcloudconf.raml

```diff
@@ -100,8 +93,8 @@ traits:
               {
                 "rt": ["oic.r.coapcloudconf"],
```

Comment:
Out of curiosity, why is "rt" listed but not "if"?  I would also expect oic.if.rw to return at, apn and cis but I'm not sure why "rt" should be there...

Reply:
Have removed "rt" in next PR in response example of "rw" interface. But not sure it matters from example perspective. AFAIK Implementation does return as of now. I think we had similar discussion during WES.